modules/ding_news/ding_news.pages_default.inc

```diff
@@ -994,7 +1018,7 @@ news/%tid:name',
   $display->title_pane = 'new-bb26b1db-7e7e-4c3f-8956-183ed5e31146';
   $handler->conf['display'] = $display;
   $page->default_handlers[$handler->name] = $handler;
-  $pages['ding_news'] = $page;
+  $pages[''] = $page;
```

Comment:
see below

Reply:
This is an error caused by the features module after a field-group update.